src/main/java/com/google/cloud/tools/app/impl/appcfg/AppCfgAppEngineStandardStaging.java

```diff
@@ -82,16 +82,17 @@ public void stageStandard(StageStandardConfiguration configuration) throws AppEn
       arguments.add("--enable_jar_classes");
     }
 
-    Path dockerfile =
-        configuration.getDockerfile() == null ? null : configuration.getDockerfile().toPath();
-    Path dockerfileDestination = configuration.getStagingDirectory().toPath();
+    File dockerfile =
+        configuration.getDockerfile() == null ? null : configuration.getDockerfile();
+    File dockerfileDestination = configuration.getStagingDirectory();
 
     try {
 
       appEngineSdk.runCommand(arguments);
 
       if (dockerfile != null && dockerfileDestination != null) {
-        Files.copy(dockerfile, dockerfileDestination, StandardCopyOption.REPLACE_EXISTING);
+        Files.copy(dockerfile.toPath(), dockerfileDestination.toPath(),
```

Comment:
It would be nice, but I'm not aware of a simple API in the JDK for copying files without using Paths.